### PR TITLE
Starter Application Setup

### PR DESCRIPTION
**Pull request by timschmidtdev**
@droberts84
Hi David, 

I completed the Bloc Starter Application workflow for BlocChat.
index.html should now connect to Angular, AngularFire and Firebase.

### CODE REVIEW

File: app/pages/index.html [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
     <meta charset="UTF-8">
     <title>Bloc Base Project</title>
     <link rel="stylesheet" href="/styles/style.css">
 </head>
 <body>
     <script src="/scripts/app.js"></script>
+    <!-- Angular -->
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.8/angular.min.js"></script>
+    <!-- AngularFire -->
+    <script src="https://cdn.firebase.com/libs/angularfire/2.0.1/angularfire.min.js"></script>
```
Comment (drobtravels) [inline app/pages/index.html:13]: Angular Fire should be after Firebase, since its really just a wrapper for Firebase

File: app/pages/index.html [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
+    <!-- AngularFire -->
+    <script src="https://www.gstatic.com/firebasejs/3.7.3/firebase.js"></script>
```
Comment (drobtravels) [inline app/pages/index.html:15]: You may want to use 3.6.6 due to [this issue](https://github.com/firebase/angularfire/issues/922)  I'm not sure if its been resolved in 3.7.3

**Review by drobtravels — CHANGES_REQUESTED** [review]
All good other that the comments I made

File: app/pages/index.html (hunk 2 above)
Comment (timschmidtdev) [reply app/pages/index.html:15]: Whoa. That's a big issue. I'll make the change to 3.6.6

File: app/pages/index.html (hunk 1 above)
Comment (timschmidtdev) [reply app/pages/index.html:13]: I hear what your saying. I'll make the update. 
Check out the instructions from Bloc. About half way down the page, it shows Angular, AngularFire, and Firebase in this order.

https://www.bloc.io/resources/getting-started-with-frontend-projects

**Pull request merged**